docs/pages/branding/careers.tsx

```diff
@@ -0,0 +1,491 @@
+        borderColor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.400' : 'grey.400'),
+      }}
+    >
+      <Typography component="div" variant="body2" fontWeight="bold" sx={{ mb: 1 }}>
+        {title}
+      </Typography>
+      {children}
+      <Link href={url} variant="body2">
+        {linkText || 'Learn more'} <KeyboardArrowRightRounded fontSize="small" sx={{ mt: '1px' }} />
+      </Link>
+    </Paper>
+  );
+};
+
+const Question = ({
+  question,
+  children,
+  defaultShowAnswer = false,
+}: {
+  question: string;
+  children: React.ReactNode;
+  defaultShowAnswer?: boolean;
+}) => {
+  const [showAnswer, setShowAnswer] = React.useState(defaultShowAnswer);
+  return (
+    <Paper variant="outlined" sx={{ height: '100%', p: 2 }}>
+      <div style={{ width: '100%', display: 'flex', justifyContent: 'space-between' }}>
+        <Typography component="div" variant="body2" fontWeight="bold">
+          {question}
+        </Typography>
+        <span
+          role="button"
+          onClick={() => setShowAnswer(!showAnswer)}
+          onKeyDown={() => setShowAnswer(!showAnswer)}
+          tabIndex={0}
+        >
+          {showAnswer ? (
+            <KeyboardArrowUpRounded color="primary" fontSize="small" sx={{ mt: '1px' }} />
+          ) : (
+            <KeyboardArrowDownRounded color="primary" fontSize="small" sx={{ mt: '1px' }} />
+          )}
+        </span>
+      </div>
+      {showAnswer && children}
+    </Paper>
+  );
+};
+
+const Role = ({
+  title,
+  description,
+  url,
+}: {
+  title: string;
+  description: string;
+  url?: string;
+}) => (
+  <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+    <div>
+      <Typography variant="body2" color="text.primary" fontWeight={600} sx={{ my: 1 }}>
+        {title}
+      </Typography>
+      <Typography color="text.secondary" sx={{ mb: 1, maxWidth: 450 }}>
+        {description}
+      </Typography>
+    </div>
+    {url && (
+      <div style={{ display: 'flex', alignSelf: 'start', paddingTop: '0.5em' }}>
+        <Link href={url} variant="body2">
+          More about this role <KeyboardArrowRightRounded fontSize="small" sx={{ mt: '1px' }} />
+        </Link>
+      </div>
+    )}
+  </Box>
+);
+
+function CareersContent() {
+  return (
+    <React.Fragment>
+      {/* Hero */}
+      <Container>
+        <Box
+          sx={{
+            height: '40vh',
+            minHeight: 300,
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'center',
+            alignItems: 'center',
+            maxWidth: 600,
+            mx: 'auto',
+            textAlign: 'center',
+          }}
+        >
+          <Typography variant="body2" color="primary.600" fontWeight="bold">
+            Careers
+          </Typography>
+          <Typography component="h1" variant="h2" sx={{ my: 1 }}>
+            Build the <GradientText>next generation</GradientText> of tools for UI development.
```

Comment:
```suggestion
            Build the <GradientText>next generation</GradientText> of tools for UI development
```

Just removing the period 😅